packages/fxa-content-server/server/lib/routes/react-app/index.js

```diff
@@ -9,6 +9,14 @@ const { TERMS_PRIVACY_REGEX } = require('./content-server-routes');
  * group object it should go in and add a new object in `routes` by calling `.getRoute`
  * or setting `routes` with `.getRoutes` on the react route object.
  *
+ * If a routeGroup should always be rendered with React in production, set `fullProdRollout` to true.
+ * These routes can still be turned off on SRE side by switching featureFlagOn to false
+ * on stage or prod for the relevant route if needed. React will ALWAYS be off for the routeGroup
+ * if featureFlagOn is set to false.
+ *
+ * NOTE however that routes that have been sunset in content-server (i.e, simple routes) can no longer
```

Comment:
👍

Did you have any luck here running `tests/server/routes/react-app.js`? I feel like we could use one test for the `fullProdRollout` addition, but I just looked that file over and am seeing they do a lot of testing of the "exclude" functions within files. It might be best to have a playwright test for this, but then it's tricky if we've deleted Backbone files... maybe we need one route that we're going to deem we're keeping around for testing seeing Backbone v React. I won't block on this.

Reply:
I have not been able to run `tests/server/routes/react-app.js`, it will need https://github.com/mozilla/fxa/pull/16044 to re-enable the server tests.